packages/swap/prisma/schema.prisma

```diff
@@ -490,10 +490,14 @@ model Quote {
   quotedIntermediateAmount            Decimal?           @db.Decimal(30, 0)
   quotedEgressAmount                  Decimal            @db.Decimal(30, 0)
   quotedPrice                         Decimal
+  inputAssetPriceAtChannelOpening     Decimal?
```

Comment:
i think storing the `indexPriceAtChannelOpening`/`indexPriceAtCompletion`  could be convenient bc you can compare that one directly with `quotedPrice`/`executedPrice`

Reply:
yeah i'll add it